source/documentation/doc-Introduction_to_the_VM_Portal/topics/Connecting_to_virtual_machines.adoc

```diff
@@ -0,0 +1,18 @@
+=== Connecting to virtual machines
+
+*Connecting to a virtual machine*
+
+. In the virtual machines pane, click the *Run* button in a virtual machine's card to start that virtual machine.
+
+. Click the *Console* button to connect to the virtual machine.
+
+. You will be asked to download a *.vv* file.
+
+. Open the file with *remote-viewer*. A console window displays.
++
+You can now use the virtual machine in the same way you would use a physical desktop.
+
+[NOTE]
+====
+To allow other users to connect to the VM, make sure you restart the virtual machine when you are finished using the console.
```

Comment:
I see that the error that is returned by the system also include the option of setting "Disable strict user checking":
`Console connection denied. Another user has already accessed the
console of this VM. The VM should be rebooted to allow another user to
access it, or changed by an admin to not enforce reboot between users
accessing its console.`
I think we should mention this option here as well